docs/source/api/plugin/usage-reporting.md

```diff
@@ -105,6 +105,38 @@ The only properties of the reported error you can modify are its `message` and i
 <tr>
 <td>
 
+###### `fieldLevelInstrumentation`
+
+`async Function` or `number`
```

Comment:
or `false`?

Reply:
Hmm, I don't think so.

A number is a short-hand for `() => Math.random() <= n ? 1/n : 0`.  `false` would just be shorthand for `() => false`.  I think writing the latter yourself is easy enough.